packages/prismic/tests/usePrismic.spec.ts

```diff
@@ -107,6 +112,10 @@ describe('[prismic] usePrismic', () => {
   });
 
   it('should have correct initial values', () => {
+    mockedUtils.useSSR.mockReturnValueOnce({
+      initialState: null,
+      saveToInitialState: jest.fn()
+    });
```

Comment:
Since this is repeated literally in every test case, wouldn't it be better to move it to the `beforeEach` function above?

Moreover - I think there should be at least one additional test case for when we have state stored in the initial state.

Reply:
1.  agree, but then of course `mockReturnValueOnce` needs to be replaced by `mockReturnValue`
2. agree too, but it is up to you @EugeneHlushko